docs/source/release/v6.6.0/Indirect/New_features/Used/34636.rst

```diff
@@ -0,0 +1 @@
+- Change "Group by sample" option to "Sample changer grouped" in ISIS Energy Transfer interface and only show it when IRIS instrument is selected
```

Comment:
I think this should not be in used. Isn't that right @ConorMFinn?

Reply:
That's correct. Release notes go into `Used` once they've been amalgamated into the main files.